README.md

```diff
@@ -1,18 +1,17 @@
 # Mattermost Handbook
 
-The purpose of this handbook is to _**iteratively increase clarity**_ for Mattermost staff and community co-building the future of our company and our offerings.
+The Mattermost Handbook is a continual work-in-progress for _Mattermost staff_--people who are paid to work on our open source software and to help run the company behind our software. The purpose of this handbook is to _**iteratively increase clarity**_ for Mattermost staff and community co-building the future of our company and our offerings.
```

Comment:
It's not just for paid staff. There are handbook entries for the contributor community as well, for instance: https://handbook.mattermost.com/contributors/contributors/ways-to-contribute